build/Dockerfile

```diff
@@ -1,7 +1,19 @@
-FROM alpine:3.8
+#
+# Dockerfile for building local images.
+#
 
-RUN apk upgrade --update --no-cache
+# Build the manager binary
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.10 as builder
```

Comment:
You can also remove this. You can make a combination of the image layers that you want in your ci-operator configuration.